scenarios/invalid_v_column/README.md

```diff
@@ -0,0 +1,28 @@
+# Workflow: Scenario (invalid v column)
+
+## What is the purpose of this scenario?
+The current workflow operators do not provide a means to to prevent including v clumn when we executing SELECT * in a Hive query. Preventing the inclusion of the v column is a required in certain cases, for example when recreating a table. This example show how to do it by using http>: operator to call Treasure Data API.
```

Comment:
The current workflow operators do not provide a means to prevent including v column when executing SELECT * in a Hive query. Preventing the inclusion of the v column is required in certain cases, for example, when recreating a table. This example shows how to exclude the v column by using the http>: operator to call Treasure Data API.